### DISCUSSION TITLE
HTTP error 401

### DISCUSSION THREAD

**Discussion by AnnaKinghorn** [IBAHCM/RPiR#551] [Q&A for technical issues] [answered]
Hi Richard,

Happy New Year :) I've been having a problem with the command devtools::install() and was wondering if you could help?
It has been outputting an error saying:

Error: HTTP error 401.
  Bad credentials

  Rate limit remaining: 58/60
  Rate limit reset at: 2023-01-04 14:01:21 UTC

I was wondering if you know what the issue is here as I can't seem to work it out?
Many thanks,
Anna

**Comment by richardreeve** [comment, marked as answer]
Hi @AnnaKinghorn - it looks like your Personal Access Token on GitHub has probably expired. I imagine you probably can't push anything to any of of your repositories as well. I'd try going through these instructions again to create a new PAT and get RStudio to recognise it - https://sbohvm.github.io/RPiR/articles/pages/install_github.html#add-a-personal-access-token-pat - you'll need to replace the PATs rather than create them from scratch, but the instructions are the same. Let me know if anything isn't clear (or that doesn't help!).

**Reply by AnnaKinghorn** [reply]
Hi Richard,
Yes, that's exactly what happened - it's working fine now. Thank you!
